tests/testthat/test-connect_lamindata.R

```diff
@@ -0,0 +1,58 @@
+test_that("Connecting to lamindata works", {
```

Comment:
I would probably add `skip_if_offline()` to avoid failures later

Reply:
Good point!

```suggestion
skip_if_offline()

test_that("Connecting to lamindata works", {
```